lib/chusaku/cli.rb

```diff
@@ -41,7 +41,8 @@ def call(args = ARGV)
     # @raise [Chusaku::CLI::NotARailsProject] Exception if not Rails project
     # @return [void]
     def check_for_rails_project
-      has_controllers = File.directory?("./app/controllers")
+      controllers_pattern = options[:controllers_pattern] || DEFAULT_CONTROLLERS_PATTERN
+      has_controllers = Dir.glob(Rails.root.join(controllers_pattern)).present?
```

Comment:
Ah, thanks for adding this in! This was something that I overlooked when we were adding this flag in. I don't use engines very often, but I'm guessing most projects that use Rails engines normally have at least one controller in the default Rails app? Is it possible for a Rails project to use _only_ engines and not having anything in the top-level Rails app?

Reply:
It is possible. We use [packs-rails](https://github.com/rubyatscale/packs-rails), and have all our controllers in an embedded engine. That's probably not a common case though.

Reply:
Got it. I'm wondering if it's worth considering that one-off use case for this PR. If Chusaku starts supporting engines, it could make sense to have it support a _just_ engines setup as well. When I first wrote this gem, I was working under the assumption that `app/controllers/*` will always exist, but perhaps there's a better and more simple way to just check that the project is a Rails project.